shared/actions/json/fs.json

```diff
@@ -71,6 +71,12 @@
     "uninstallKBFSConfirm": {
       "onSuccess": "() => void"
     },
-    "uninstallKBFS": {}
+    "uninstallKBFS": {},
+    "fsActivity": {},
+    "syncingStatus": {
```

Comment:
Is there a reason you need a new action for this? Can't you just use `setFlags`? We have a few extra actions that affect state flags only because we also need a Saga for those actions.

Reply:
The flags PR wasn't merged when this PR was made, so that didn't came to my mind initially. I'll try to use `setFlags`.